src/util/ICM_20948_C.c

```diff
@@ -2,250 +2,346 @@
 #include "ICM_20948_REGISTERS.h"
```

Comment:
This is me picking nits - these whitespace changes make it hard to see if any significant code changes happened in the C implementation.

It's a personal thing but I'm not a big fan of VSC's style when it comes to automatic formatting. 

I wonder how much effort it will take to revert the whitespace changes w/o undoing any important code changes.

Reply:
Same goes for the C.h file but I'll keep the comment here